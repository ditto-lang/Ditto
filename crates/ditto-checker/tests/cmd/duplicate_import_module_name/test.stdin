module Test exports (..)

import (foo) A import (bar) A
