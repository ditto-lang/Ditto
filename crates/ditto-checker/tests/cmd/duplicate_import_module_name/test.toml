bin.name = "ditto-checker-testbin-check-module"
args = ["duplicate_import_module_name.ditto"]
fs.sandbox = true
status = "failed"
