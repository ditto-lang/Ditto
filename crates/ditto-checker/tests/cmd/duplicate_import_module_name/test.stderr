
  × duplicate imports for module `A`
   ╭─[duplicate_import_module_name.ditto:2:1]
 2 │ 
 3 │ import (foo) A import (bar) A
   ·              ┬              ┬
   ·              │              ╰── imported again here
   ·              ╰── previous import
   ╰────
  help: try aliasing one of the imports?

